index.js

```diff
@@ -167,6 +169,23 @@ bot.on('ready', () => {
             event);
       }
     });
+
+    schedule.scheduleJob('*/5 * * * *', () => {
+      bot.getMessages({ channelID: DEV_NULL_CHANNEL, limit: 100 }, (err, messages) => {
```

Comment:
Maybe chuck this in a loop to combat spam, otherwise > 100 msgs per 5min wins (until the pace slows) Probably not 100% necessary, just being nitpicky.